### PR TITLE
File submission on frontend to backend

### PR DESCRIPTION
**Pull request by aaronliuo**
Rearranged the create form onSubmit function to submit problem data and test case files to the backend properly. There are three axios requests onSubmit now. First axios request essentially gets the last post id so it can assign the next problem's id. Second request sends the problem data. Third request sends the files.

### CODE REVIEW

File: frontend/src/pages/create.js [hunk 1]
```diff
@@ -47,6 +47,7 @@ const Create = () => {
   }
   
   const UserContext = createContext()
+  const [file, setFile] = useState(null);
```
Comment (Giovanni1014) [inline frontend/src/pages/create.js:50]: I don't think we need file to be a state cuz we don't need it rendered every time it changes.

**Review by Giovanni1014 — CHANGES_REQUESTED** [review]
unnecessary use of state

File: frontend/src/pages/create.js (hunk 1 above)
Comment (aaronliuo) [reply frontend/src/pages/create.js:50]: Okay, Ill look into changing it

**Review by Giovanni1014 — APPROVED** [review]
Great work

**Review by r4c3 — APPROVED** [review]
approved by default pr30

**Pull request merged**